release-notes/v1_6.md

```diff
@@ -7,70 +7,88 @@ MetaDescription: See what is new in the Visual Studio Code September 2016 Releas
 
 # September 2016 (version 1.6)
 
-This release has some big infrastructure changes for Visual Studio Code. We've updated to the latest Electron shell (version 1.3.7) and now include TypeScript 2.0.  We added programmatic access to user and workspace settings and you'll see this is action in several places in the UI (**File** > **Auto Save**, **Configure File Associations**). And we're continuing to improve Extension management (**Configure Workspace Recommended Extensions**, support for installing an **Extension Pack**). All of this work puts VS Code in great shape for us and our extension partners moving forward.
+This release has some big infrastructure changes for Visual Studio Code and several useability imporvements. 
 
-Here are some other highlights:
 
-* **Workbench**: File Icons added to Open Editors, Search Results, Problems, and Tabs. Automatic detection of corrupted installation.
-* **Editor**: Format On Save, Vim style relative line numbers.
-* **Languages**: TypeScript 2.0 language features which benefits both TypeScript and JavaScript development.
-* **Extensions**: Create workspace recommended Extension lists. Extension packs to automatically include dependencies.
-* **Debugging**: Support for running NPM and other script runners on launch. 
-* **Extension Authoring**: New APIs to update settings and modify files before save. Debug protocol updates including programmatic access to `launch.json`.
+![1.6 Release hightlights include: TypeScript 2.0, Release notes within VS Code, Format on Save, Swich between Windows, Search history, run scripts on debugger launch, workspace extension recomendations, extensiosn packs, Extensions updates for JS/TS colorizors, VIM, Experimental Node Debugger and a new API for settings access](/release-notes/images/1_6/ReleaseHighlights_1.6.png)
```

Comment:
I'm keen on the image only introduction. It isn't accessible, as you say doesn't tell a story and I find the purple with yellow text jarring. I think the banner is great for a tweet where you don't have much real estate but I don't know how much people can get out of short four or five word descriptions.